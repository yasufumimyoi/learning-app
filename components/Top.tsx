import { CheckIcon } from '@heroicons/react/outline'

const Top = () => (
  <div className="mb-10">
    <div className="flex items-center mb-3">
      <CheckIcon className="h-5 w-5 text-green-500 mr-2" />
      <p className="text-white text-xs md:text-base">
        チュートリアルは終わったけど、次に何を学習したらいいのか分からない...
      </p>
    </div>
    <div className="flex items-center mb-3">
      <CheckIcon className="h-5 w-5 text-green-500 mr-2" />
      <p className="text-white text-xs md:text-base">
        基礎は学習したけど、何を作っていいのか分からない...
      </p>
    </div>
    <div className="flex items-center mb-3">
      <CheckIcon className="h-5 w-5 text-green-500 mr-2" />
      <p className="text-white text-xs md:text-base">
        そんなアナタにおすすめのMission (動画) をPick Upしました！
      </p>
    </div>
  </div>
)

export default Top
