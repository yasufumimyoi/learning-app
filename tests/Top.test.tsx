import { render } from '@testing-library/react'
import Top from '../components/Top'

it('Topのsnapshot"', () => {
  const component = render(<Top />)
  expect(component).toMatchSnapshot()
})
