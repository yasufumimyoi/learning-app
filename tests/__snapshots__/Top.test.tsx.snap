// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Topのsnapshot" 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="mb-10"
      >
        <div
          class="flex items-center mb-3"
        >
          <svg
            class="h-5 w-5 text-green-600 mr-2"
            fill="none"
            stroke="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M5 13l4 4L19 7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
          </svg>
          <p
            class=" text-xs md:text-base"
          >
            チュートリアルは終わったけど、次に何を学習したらいいのか分からない...
          </p>
        </div>
        <div
          class="flex items-center mb-3"
        >
          <svg
            class="h-5 w-5 text-green-600 mr-2"
            fill="none"
            stroke="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M5 13l4 4L19 7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
          </svg>
          <p
            class="text-xs md:text-base"
          >
            基礎は学習したけど、何を作っていいのか分からない...
          </p>
        </div>
        <div
          class="flex items-center mb-3"
        >
          <svg
            class="h-5 w-5 text-green-600 mr-2"
            fill="none"
            stroke="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M5 13l4 4L19 7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
          </svg>
          <p
            class=" text-xs md:text-base"
          >
            そんなアナタにおすすめのMission (動画) をPick Upしました！
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="mb-10"
    >
      <div
        class="flex items-center mb-3"
      >
        <svg
          class="h-5 w-5 text-green-600 mr-2"
          fill="none"
          stroke="currentColor"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M5 13l4 4L19 7"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
          />
        </svg>
        <p
          class=" text-xs md:text-base"
        >
          チュートリアルは終わったけど、次に何を学習したらいいのか分からない...
        </p>
      </div>
      <div
        class="flex items-center mb-3"
      >
        <svg
          class="h-5 w-5 text-green-600 mr-2"
          fill="none"
          stroke="currentColor"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M5 13l4 4L19 7"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
          />
        </svg>
        <p
          class="text-xs md:text-base"
        >
          基礎は学習したけど、何を作っていいのか分からない...
        </p>
      </div>
      <div
        class="flex items-center mb-3"
      >
        <svg
          class="h-5 w-5 text-green-600 mr-2"
          fill="none"
          stroke="currentColor"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M5 13l4 4L19 7"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
          />
        </svg>
        <p
          class=" text-xs md:text-base"
        >
          そんなアナタにおすすめのMission (動画) をPick Upしました！
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
